### PR TITLE
MMF-1123 Quality Widgets

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

**Comment by gregaubert** [comment]
@henryju If you can just have a quick look just in case you can spot anything weird :)

File: gulpfile.js [hunk 1]
```diff
@@ -191,7 +191,11 @@ gulp.task('extension:copy', () =>
   es
     .merge(
       gulp.src(
-        path.join(paths.extensions.root, '**', '@(vss-extension.json|extension-icon.png|*.md)')
+        path.join(
+          paths.extensions.root,
+          '**',
+          '@(vss-extension.json|extension-icon.png|*.md|*.html)'
```
Comment (henryju) [inline gulpfile.js]: I don't see any html file added to the extension, so why do you need to add that?

File: extensions/sonarcloud/vss-extension.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "type": "git",
     "uri": "https://github.com/SonarSource/sonar-scanner-vsts"
   },
+  "baseUri": "https://sonarcloud.io",
```
Comment (henryju) [inline extensions/sonarcloud/vss-extension.json]: Have you checked it doesn't break other resources (like icons or tasks)?

File: gulpfile.js (hunk 1 above)
Comment (gregaubert) [reply gulpfile.js]: Ah thanks it's a leftover from one of my early testing!

File: extensions/sonarcloud/vss-extension.json (hunk 2 above)
Comment (gregaubert) [reply extensions/sonarcloud/vss-extension.json]: I'll check again to be sure! Thanks!

Comment (gregaubert) [reply extensions/sonarcloud/vss-extension.json]: Seems to be all good!

**Pull request merged**